build_package.py

```diff
@@ -1234,6 +1233,31 @@ def create_scriptlets():
     fp.write(
         r"""#!/bin/bash
 
+echo "Check for python."
+# look for 'python' by '/usr/bin/env', exit with error if it is not found.
+/usr/bin/env python --version >/dev/null 2>&1 || \
```

Comment:
I believe I had this comment on another PR or Slack - I would prefer to store all those scripts inside files (e.g. ``installer/scripts/{preinstall.sh,preuninstall.sh}`` and re-use them here instead of storing the script content inline.

Imo, this makes it more organized, easier to test in a standalone fashion and also gives us flexibility to re-use those scripts if we ever move away from build_package.py and fpm tp some other mechanism

Reply:
Yes, I agree, and I think there is more that can be changed in `build_package.py` script.